main.py

```diff
@@ -530,6 +530,10 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'%s/resubmit/<suggestion_id>' % feconf.SUGGESTION_ACTION_URL_PREFIX,
         suggestion.ResubmitSuggestionHandler),
+    get_redirect_route(
```

Comment:
Move it to line 526 (Alphabetical order).

Reply:
done